src/index.html

```diff
@@ -20,6 +20,9 @@ <h1>Harris is Harry Potter</h1>
     <footer class='Footer'>
       <p class='Footer-contact'>For comments, questions, or general enquiries, contact us at <a href='mailto:hello@harrisisharrypotter.com'>hello@harrisisharrypotter.com</a>.</p>
     </footer>
+    <div class='Harris'>
+      <img src='./harris-potter.png' />
```

Comment:
Can we lazy-load this image after the "Konami code" has been entered? Most people won't ever see this, so there's no reason for their browser to download the image.